src/dbnode/storage/bootstrap/bootstrapper/commitlog/source.go

```diff
@@ -146,6 +146,7 @@ func newCommitLogSource(
 func (s *commitLogSource) AvailableData(
 	ns namespace.Metadata,
 	shardsTimeRanges result.ShardTimeRanges,
+	_ bootstrap.State,
```

Comment:
nit: Seems like `AvailableData` and `AvailableIndex` share the same arguments. Might be cleaner to refactor into an options struct?

Reply:
Yeah, I think that's a good idea but probably going to punt on it simply to avoid having to fix so many tests from changing the interface.